package.json

```diff
@@ -19,6 +19,7 @@
   },
   "homepage": "https://github.com/opentok/rtcstats-server#readme",
   "dependencies": {
+    "@google-cloud/bigquery": "^4.5.0",
```

Comment:
I'm not sure which is the best way to tackle it, but would be nice to install gcp/aws explicitely, so users can choose the backend they need.

For instance using [optional dependencies](http://npm.github.io/using-pkgs-docs/package-json/types/optionaldependencies.html)?

Reply:
optional dependencies would only work if we made a meta package that requires rtcstats-server and install bigquery et al. The overhead isn't so big i think